src/screens/Customers/components/MeterInfoContainer.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { createStyles, withStyles, Theme } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import { CardPropsInfo } from '../../../components/OutlinedCardList';
+import IconButton from '@material-ui/core/IconButton';
+import { Create as CreateIcon } from '@material-ui/icons';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      width: '50%',
+      display: 'flex',
+    },
+    info: {
+      width: '100%',
+      marginBottom: '8px',
+      padding: '0px 12px 4px',
+      border: '1px solid',
+      height: '60px',
+      maxHeight: '60px',
```

Comment:
Why do you have height and maxHeight?

Reply:
Also in general, avoid setting set pixel heights, aside from `minHeight`. If I shrink my screen width, you can see how `OutlinedCardList` adapts the height whereas this overflows.

![image](https://user-images.githubusercontent.com/21160510/114234610-dd4b1800-9933-11eb-9060-377956c362fd.png)

Reply:
to make sure the grid format stayed the same. also would people really view the app at such a small screen width?